test/e2e/tests/crtitical_tests_prs/test_community_create_edit_leave.py

```diff
@@ -17,15 +17,29 @@
 @allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/703057', 'Edit community')
 @pytest.mark.case(703057)
 @pytest.mark.critical
-def test_edit_community(main_screen: MainWindow):
+def test_create_edit_leave_community(main_screen: MainWindow):
     with step('Enable creation of community option'):
         settings = main_screen.left_panel.open_settings()
         settings.left_panel.open_advanced_settings().enable_creation_of_communities()
 
-    with step('Create community and select it'):
+    with step('Open create community popup'):
+        communities_portal = main_screen.left_panel.open_communities_portal()
+        create_community_form = communities_portal.open_create_community_popup()
+
+    with step('Verify fields of create community popup and create community'):
         community = RandomCommunity()
-        main_screen.create_community(community_data=community)
-        community_screen = main_screen.left_panel.select_community(community.name)
+        community_screen = create_community_form.create_community(community_data=community)
+
+    with step('Verify community parameters in community overview'):
+        assert community_screen.left_panel.name == community.name
```

Comment:
Verify other community params, like banner/logo/tags? (Assuming you're setting them elsewhere too)

Reply:
@caybro yes, that is next thing to do, will do separately